Src/IronPython/Lib/iptest/type_util.py

```diff
@@ -44,7 +52,8 @@ def remove_clr_specific_attrs(attr_list):
     
     # CLR array shortcut
     array_cli       = System.Array
-    array_int       = System.Array[int]
+    array_int       = System.Array[System.Int32]
```

Comment:
I know, me too. I didn't notice [Upgrading from IronPython 2 to 3](https://github.com/IronLanguages/ironpython3/blob/master/Documentation/upgrading-from-ipy2.md#upgrading-from-ironpython-2-to-3) before. It will put a comprehensive note there (separate PR).